conf/server/server.conf

```
@@ -17,3 +17,13 @@ server {
     # db_conn_string: postgres database connection string.
     db_conn_string = "postgresql://postgres:postgres@localhost:5432/galadriel"
 }
+
+providers {
+    # X509CA "disk": Uses a ROOT CA loaded from disk to issue X509 certificates.
+    x509ca "disk" {
+        # Path to the root CA private key file. PEM format.
```

Shouldn't we specify here that we're expecting RSA keys?
```suggestion
        # Path to the root CA private RSA key file. PEM format.
```

---

It can be both EC and RSA keys.